Source/Extensions/Blazorise.RichTextEdit/JSRichTextEditModule.cs

```diff
@@ -60,15 +63,15 @@ public async ValueTask<IAsyncDisposable> Initialize( RichTextEdit richTextEdit )
     {
         var dotNetRef = DotNetObjectReference.Create( richTextEdit );
 
-        await InvokeSafeVoidAsync( "initialize",
-            dotNetRef,
-            richTextEdit.ElementRef,
-            richTextEdit.ElementId,
-            richTextEdit.ReadOnly,
-            richTextEdit.PlaceHolder,
-            richTextEdit.Theme == RichTextEditTheme.Snow ? "snow" : "bubble",
-            richTextEdit.SubmitOnEnter,
-            richTextEdit.ConfigureQuillJsMethod );
+        await InvokeSafeVoidAsync( "initialize", dotNetRef, richTextEdit.ElementRef, richTextEdit.ElementId, new RichTextEditJSOptions
```

Comment:
Finally!! A Typed options class instead of anonymous, love this.

Reply:
haha